core/src/main/resources/error/error-classes.json

```
@@ -177,6 +177,11 @@
     ],
     "sqlState" : "22018"
   },
+  "CREATE_PERSISTENT_OBJECT_OVER_TEMP_OBJECT" : {
```

Could you make it shorter? like
```suggestion
  "CANNOT_REFER_TO_TEMP_OBJ" : {
```